HRIS.Services/Services/DashboardService.cs

```diff
@@ -0,0 +1,247 @@
+﻿using HRIS.Models;
+using HRIS.Services.Interfaces;
+using RR.UnitOfWork;
+using RR.UnitOfWork.Entities.HRIS;
+
+namespace HRIS.Services.Services
+{
+    public class DashboardService : IDashboardService
+    {
+        private readonly IUnitOfWork _db;
+
+        public DashboardService(IUnitOfWork db)
+        {
+            _db = db;
+        }
+        public async Task<ChurnRateDataCardDto> CalculateEmployeeChurnRate()
```

Comment:
```suggestion
        }
        
        public async Task<ChurnRateDataCardDto> CalculateEmployeeChurnRate()
```
Requires line spacing between constructor and method